src/content/docs/en/guides/astro-db.mdx

```diff
@@ -130,11 +131,13 @@ const Comment = defineTable({
 });
 ```
 
-## Seed your database
+## Seed your database for development
 
-In development, Astro will use your DB config to generate local types according to your schemas. These will be generated fresh each time the dev server is started, and will allow you to query and work with the shape of your data with type safety and autocompletion.
+In development, Astro will use your DB config to generate local types according to your schemas. These will be generated fresh from your seed file each time the dev server is started, and will allow you to query and work with the shape of your data with type safety and autocompletion.
 
-To seed development data for testing and debugging into your Astro project, create a `db/seed.ts` file. Import both the `db` object and any configured table from `astro:db`. Use the `db.insert()` function to provide an array of table row data objects.
+You will not have access to production data during development unless you [seed from your remote database](#seed-from-your-remote-database). This protects your data while allowing you to test and develop with type-safety using simulated data.
+
+To seed development data for testing and debugging into your Astro project, create a `db/seed.ts` file. Import both the `db` object and any configured table from `astro:db`. Use the `db.insert()` function to provide an array of table row data objects. This development data should match the form of your database schema and production data.
```

Comment:
> Use the `db.insert()` function to provide an array of table row data objects.

This sentence seems weird. And it is technically incorrect since the `insert()` function doesn't take any row data; those go into the `values()` function on the returned object.

Maybe we should be more generic:
```suggestion
To seed development data for testing and debugging into your Astro project, create a `db/seed.ts` file. Import both the `db` object and any configured table from `astro:db`. Use the `db` handle (the same as given to your application) to insert the initial data into the development DB. This development data should match the form of your database schema and production data.
```